src/components/serializedChallenge.ts

```diff
@@ -1,4 +1,5 @@
 import simpleTypeGuard, { SimpleArray, SimpleStringOptional, SimpleNumber, SimpleString, SimpleBooleanOptional, SimpleObjectOptional, SimpleSkip } from 'simple-type-guard';
+import { ACTOR } from './creator/Editor/SceneEdition/mapUtils';
```

Comment:
Hagamoslo todo con las constantes no ?

```suggestion
import { ACTOR, EMPTY } from './creator/Editor/SceneEdition/mapUtils';
```